src/main/js/src/containers/ErrorBoundary/ErrorBoundary.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
```

Comment:
A small thing but I like to leave an empty line between imports that are from dependencies and ones that are of our doing.